utils/build/rollup.config.js

```diff
@@ -99,8 +94,20 @@ export default [
       !!process.env.ANALYZE_BUNDLE && analyze({ summaryOnly: true })
     ],
     output: [
-      { file: pkg.main, format: 'cjs', interop: 'auto' },
-      { file: pkg.module, format: 'es' }
+      {
+        file: pkg.main,
+        format: 'cjs',
+        interop: 'auto',
+        banner: BANNER // Apply global Zendesk license
+      },
+      {
+        dir: path.dirname(pkg.module),
+        format: 'es',
+        preserveModules: true,
+        preserveModulesRoot: 'src',
```

Comment:
[preserveModulesRoot](https://rollupjs.org/configuration-options/#output-preservemodulesroot) is needed to maintain the default folder structure for cases when a component imports a dependency that isn't "externalized". Ex: `ChevronButton` inlines SVG import. 

### Default `/dist` folder structure

```
.
└── dist/
    ├── esm/
    │   ├── index.js
    │   ├── elements
    │   ├── styled
    │   ├── types
    │   └── node_modules (inline svg import)
    ├── index.cjs.js
    └── typings

```